includes/config.inc

```diff
@@ -454,6 +454,7 @@ function islandora_solr_metadata_management($data) {
         ),
         'weight' => array(
           '#type' => 'weight',
+          '#delta' => $delta >= 10 ? $delta : 10,
```

Comment:
I was going to suggest using `max()` here instead, but then I ran some quick tests, and y'know what? The ternary is faster. So, thanks for that, PHP